### PR TITLE
[fix](inverted index) add in list to fast execute logic after hit index

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

1. resolve the issue where the case fails when enable_common_expr_pushdown is set to false

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

File: be/src/vec/exprs/vexpr.h [hunk 1]
```diff
@@ -231,6 +231,47 @@ class VExpr {
...
+        return true;
+    }
+
+    virtual std::string gen_predicate_result_sign(Block& block, const ColumnNumbers& arguments,
+                                                  const std::string& function_name) {
+        std::string pred_result_sign;
+        std::string column_name = block.get_by_position(arguments[0]).name;
+        pred_result_sign += BeConsts::BLOCK_TEMP_COLUMN_PREFIX + column_name + "_";
+        if (function_name == "in") {
+            pred_result_sign += "in_list_";
```
Comment (xiaokang) [inline be/src/vec/exprs/vexpr.h:262]: change to consistent name

File: be/src/vec/exprs/vexpr.h [hunk 2]
```diff
@@ -231,6 +231,47 @@ class VExpr {
...
+                    result, ColumnNullable::create(std::move(result_column),
+                                                   ColumnUInt8::create(input_rows_count, 0)));
+        } else {
+            block.replace_by_position(result, std::move(result_column));
+        }
+
+        return true;
+    }
+
+    virtual std::string gen_predicate_result_sign(Block& block, const ColumnNumbers& arguments,
```
Comment (xiaokang) [inline be/src/vec/exprs/vexpr.h:256]: remove virtual

File: be/src/vec/exprs/vexpr.h [hunk 3]
```diff
@@ -231,6 +231,47 @@ class VExpr {
         return nullptr;
     }
 
+    // fast_execute can direct copy expr filter result which build by apply index in segment_iterator
+    virtual bool fast_execute(Block& block, const ColumnNumbers& arguments, size_t result,
```
Comment (xiaokang) [inline be/src/vec/exprs/vexpr.h:235]: remove virtual

**Comment by zzzxl1993** [comment]
run buildall

**Review by airborne12 — APPROVED** [review]
LGTM

**Comment by zzzxl1993** [comment]
run buildall

**Review by xiaokang — APPROVED** [review]
LGTM

**Comment by zzzxl1993** [comment]
run performance

**Comment by zzzxl1993** [comment]
run beut

**Comment by zzzxl1993** [comment]
run feut

**Comment by xiaokang** [comment]
run cloudut

**Pull request merged**